internal/job/transmitter/v1/common.go

```diff
@@ -15,20 +18,56 @@ type Common struct {
 	annotationDispatcher transmitter.AnnotationDispatcher
 	orderDispatcher      transmitter.OrderEventDispatcher
 
+	task      model.Task
+	productId string
+
 	in job.DataChan
 	sn *util.StopNotifier
 	wg *sync.WaitGroup
 }
 
+var ErrInvalidProductId = errors.New("transmit: can't parse productID")
+var ErrInvalidTaskString = errors.New("transmit: can't parse task")
+
 func NewCommon(
 	annotationDispatcher transmitter.AnnotationDispatcher,
-	orderDispatcher transmitter.OrderEventDispatcher) (*Common, error) {
-	return &Common{
+	orderDispatcher transmitter.OrderEventDispatcher,
+	params *job.UserParams) (*Common, error) {
```

Comment:
중간에 엔터 한번 넣어주시면 가독성이 더 좋을 것 같습니당

Reply:
```suggestion
func NewCommon(
	annotationDispatcher transmitter.AnnotationDispatcher,
	orderDispatcher transmitter.OrderEventDispatcher,
	params *job.UserParams)
	(*Common, error) {
```

이렇게 수정하는 게 좋을까요? 중간이 어디인지 잘 모르겠습니다.

Reply:
실수로 커밋을 눌러버린 것 같은데 suggested changes 코드 살짝 바꿔봤어요